### PR TITLE
fix: Removing unnecessary include and queries

### PR DESCRIPTION
**Pull request by kushthedude**
Partial fix for #3642

### CODE REVIEW

**Comment by kushthedude** [comment]
@iamareebjamal Please review and merge

File: app/controllers/admin/users/view/events/list.js [hunk 1]
```diff
@@ -32,11 +32,11 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name            : 'Date',
         valuePath       : 'startsAt',
-        extraValuePaths : ['endsAt'],
+        extraValuePaths : ['endsAt', 'timezone'],
```
Comment (prateekj117) [inline app/controllers/admin/users/view/events/list.js]: Undo all changes irrelevant to this issue.

**Comment by kushthedude** [comment]
There is no issue open. Its a child PR extending the recent changes to
other files.

File: app/controllers/admin/users/view/events/list.js (hunk 1 above)
Comment (iamareebjamal) [reply app/controllers/admin/users/view/events/list.js]: This change shouldn't be here

File: app/controllers/events/list.js [hunk 2]
```diff
@@ -15,10 +15,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         extraValuePaths : ['name', 'logoUrl'],
         headerComponent : 'tables/headers/sort',
         cellComponent   : 'ui-table/cell/cell-event-general',
-        options         : {
-          dateFormat: 'MMMM DD, YYYY - HH:mm A'
-        },
-        actions: {
```
Comment (iamareebjamal) [inline app/controllers/events/list.js]: Same

Comment (kushthedude) [reply app/controllers/events/list.js]: I will handle it in another PR.

Comment (iamareebjamal) [reply app/controllers/events/list.js]: OK,then revert it

Comment (kushthedude) [reply app/controllers/events/list.js]: @iamareebjamal Done, Please review and merge

File: app/controllers/events/list.js [hunk 3]
```diff
@@ -108,4 +108,4 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       isEventDeleteModalOpen : false
     });
   }
-}
```
Comment (iamareebjamal) [inline app/controllers/events/list.js]: Revert

File: app/controllers/admin/users/view/events/list.js [hunk 4]
```diff
@@ -100,6 +92,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
     } catch (e) {
       this.notify.error(this.l10n.t('An unexpected error has occurred.'));
     }
+
```
Comment (iamareebjamal) [inline app/controllers/admin/users/view/events/list.js]: Revert

File: app/controllers/admin/users/view/events/list.js [hunk 5]
```diff
@@ -118,6 +111,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       console.warn(e);
       this.notify.error(this.l10n.t('An unexpected error has occurred.'));
     }
+
```
Comment (iamareebjamal) [inline app/controllers/admin/users/view/events/list.js]: Revert

File: app/controllers/admin/users/view/events/list.js [hunk 6]
```diff
@@ -34,9 +34,9 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         valuePath       : 'startsAt',
         extraValuePaths : ['endsAt'],
         isSortable      : true,
+        width           : 170,
```
Comment (iamareebjamal) [inline app/controllers/admin/users/view/events/list.js]: Why?

File: app/controllers/admin/users/view/events/list.js [hunk 7]
```diff
@@ -34,9 +34,9 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
         headerComponent : 'tables/headers/sort',
         cellComponent   : 'ui-table/cell/cell-event-date'
-
```
Comment (iamareebjamal) [inline app/controllers/admin/users/view/events/list.js]: Revert

File: app/controllers/admin/users/view/events/list.js (hunk 6 above)
Comment (kushthedude) [reply app/controllers/admin/users/view/events/list.js]: the column was squeezed out

File: app/controllers/admin/users/view/events/list.js (hunk 7 above)
Comment (kushthedude) [reply app/controllers/admin/users/view/events/list.js]: The empty line too?

Comment (iamareebjamal) [reply app/controllers/admin/users/view/events/list.js]: Yes

**Comment by iamareebjamal** [comment]
@kushthedude Release is being made, please revert changes and finalize

**Comment by kushthedude** [comment]
chaning it right away

**Comment by kushthedude** [comment]
@iamareebjamal Done

**Comment by kushthedude** [comment]
@iamareebjamal Merge

**Pull request merged**